interface/forms/phq9/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
you can remove this file since it's linked in the file level docblock with @license that points to the project's license

Reply:
thanks @stephenwaite ,  Mark has removed the file, do we need to do another PR ?